Desktop/property-pro-app/src/components/Midsection/Midsection.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import "./midsection.css";
+
+export default function Midsection() {
+  return (
+    <div className="mid">
+      <div className="boxcontainer">
+        <div className="buy">
+          <div className="icon">
+            <img
```

Comment:
Don't use an external image url that is not in your CDN, a good solution would be to download this image and add it within your application scope.